app/models/bus_stop.rb

```diff
@@ -83,4 +84,7 @@ def assign_completion_timestamp
     assign_attributes completed_at: (completed? ? DateTime.current : nil)
   end
 
+  def check_completed
+    errors.add :base, 'You must complete all fields before marking stop as complete.'
```

Comment:
So this always adds an error for any completed stop?

I'm confused as to how there isn't any statement that checks for all of the completed fields in this PR.